src/VirtualDesktop/Interop/Proxy/IVirtualDesktopNotification.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace WindowsDesktop.Interop.Proxy;
+﻿using System.Diagnostics;
```

Comment:
It is not supposed to be included

Reply:
You're right, it was an issue on my end. Your branch is good!

Reply:
It's a problem of my branch, I'm reminding myself to remove it. Your problem might be caused by it, sorry about that!